server.js

```diff
@@ -7503,8 +7505,22 @@ cache({
             var innerText = jsonpath.join(', ');
             badgeData.text[1] = (prefix || '') + innerText + (suffix || '');
             break;
+          case 'xml':
+            data = new dom().parseFromString(data);
+            var xpathdata = xpath.select(pathExpression, data, true);
```

Comment:
Alternatively we could check `xpathdata` value and remove try-catch block. What do you think about it?
```js
let value;
if (xpathdata) {
  value = (prefix || '') + (pathExpression.indexOf('@') + 1 ? xpathdata.value : xpathdata.firstChild.data) + (suffix || '');
} else {
  value = 'no result';
  setBadgeColor(badgeData, 'lightgrey');
}
badgeData.text[1] = value;
```

Reply:
Nice, a lot more simple!